test/sample_components/test_add_value.py

```diff
@@ -2,29 +2,30 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 from sample_components import AddFixedValue
+from canals.serialization import component_to_dict, component_from_dict
 
 
 def test_to_dict():
     component = AddFixedValue()
-    res = component.to_dict()
+    res = component_to_dict(component)
     assert res == {"type": "AddFixedValue", "init_parameters": {"add": 1}}
```

Comment:
I wonder if we need all these serialization tests now that the method have been removed... wdyt?

Reply:
Indeed for the components using the default method we're basically testing the logic itself over and over. On the other end, this is how I caught the bug in the serialization logic... So no strong opinions here.